core/src/bmi.jl

```diff
@@ -56,6 +62,27 @@ function BMI.initialize(T::Type{Model}, config::Config)::Model
     return Model(integrator, config, saved_flow)
 end
 
+"Load updates from 'TabulatedRatingCurve / time' into the parameters"
+function update_tabulated_rating_curve(integrator)::Nothing
+    (; node_id, tables, time) = integrator.p.tabulated_rating_curve
+    t = time_since(integrator.t, integrator.p.starttime)
```

Comment:
`time_since` is quite confusing, as it's the inverse of `seconds_since`, but `seconds` is a measure of `time`. Would propose to rename like `unix2datetime` and `datetime2unix` (unix has an epoch in 1970, similar to our use of t0). Maybe clock2datetime? Or relsec2datetime (relative seconds)?

Reply:
I quite like the names, since:

`time_since` returns a time, and you need a since as a second argument
`seconds_since` returns seconds, and you need a since as a second argument

Reply:
I quite like the names, since:

`time_since` returns a time, and you need a since as a second argument
`seconds_since` returns seconds, and you need a since as a second argument